src/nsolid.h

```diff
@@ -27,38 +27,49 @@ namespace nsolid {
 
 class EnvInst;
 
+#define kByte "byte"
+#define kMhz "MHz"
+#define kMSecs "ms"
+#define kSecs "s"
+#define kUSecs "us"
+#define kNoUnit ""
```

Comment:
Since this is a public header, I don't love that we `#define` these more generic values that might conflict with the user's code. Possibly prepend `NS` to them or something like that as an indicator that we own them.